Improve Asgardeo Next.js quickstart (#5356)
wso2/docs-is: en/includes/quick-starts/nextjs.md

## Install Auth.js library

> Auth.js is a lightweight JavaScript library designed for simplifying authentication workflows in JavaScript web applications. The [Asgardeo provider for Auth.js](https://authjs.dev/reference/core/providers/asgardeo){:target="_blank"}  offers all the components and hooks you need to integrate your app with {{product_name}}. To get started, simply add Auth.js library to the project. Make sure to stop the dev server started in the previous step.

Reviewer: ```suggestion
Auth.js, a lightweight JavaScript library, simplifies authentication workflows in JavaScript web applications. The [Asgardeo provider for Auth.js](https://authjs.dev/reference/core/providers/asgardeo){:target="_blank"}  offers all the components and hooks you need to integrate your app with {{product_name}}. To get started, simply add Auth.js library to the project. Make sure to stop the dev server started in the previous step.
```

    pnpm dlx auth secret
    ```

> Add following entries to the `.env` or `.env.local` file, and make sure to replace the placeholders in the following code with the **`client-id`**, **`client-secret`** and **`issuer`** values you copied in **Step-1** during the application registration in the {{product_name}} console. 

Reviewer: ```suggestion
Add the following entries to the `.env` or `.env.local` file. Replace the placeholders in the following code with the **`client-id`**, **`client-secret`** and **`issuer`** values from **Step-1**.
```
